### DISCUSSION TITLE
NaturalVision Evolved in GTA Online - Compatibility Progress Thread

### DISCUSSION THREAD

**Discussion by OXP3573** [YimMenu/YimMenu#3470] [Ideas]
08/08/2024: Issue causing crash once loaded into Freemode after 1-2 minutes consistently. Pool crash again?

**Comment by maybegreat48** [comment]
If you give me the DLL/ASI files, I (or someone else) can patch them for you when I have the time

**Reply by maybegreat48** [reply]
> That's obviously not the effort of a mod developer but Rockstar dev input

No, not really. It's just a pool crash and it isn't that deep

**Reply by maybegreat48** [reply]
I've updated [YimASI](https://github.com/maybegreat48/YimASI) which should now be able to load NVE and all mods that have custom DLCs without patching anything

**Reply by OXP3573** [reply]
> Is the NVE in the room with us now?

If it was, I wouldn't need to be asking the question now, would I lol, I don't believe and in the Yim spirit there should never be an enforced shutdown of half the game... sure, ban risk is a reason cited, but as I also defended to the Discord of NVE (and got attacked for) it is user decision to test fate + not justified to charge a fee for access, essentially you would be subletting GTA Online and despite the same chat claiming Rockstar have a finger in their pie, it seems all the more reason to bypass it out of spite.

**Reply by OXP3573** [reply]
> I've updated [YimASI](https://github.com/maybegreat48/YimASI) which should now be able to load NVE and all mods that have custom DLCs without patching anything

Much appreciated, thank you for spending the time on it.

**Reply by maybegreat48** [reply]
> If it was, I wouldn't need to be asking the question now, would I lol, I don't believe and in the Yim spirit there should never be an enforced shutdown of half the game... sure, ban risk is a reason cited, but as I also defended to the Discord of NVE (and got attacked for) it is user decision to test fate + not justified to charge a fee for access, essentially you would be subletting GTA Online and despite the same chat claiming Rockstar have a finger in their pie, it seems all the more reason to bypass it out of spite.

It didn't work online because it was never meant to work online. It is not "sabotaged" or "enforced" or whatever you claim it was. Also I don't really see why people want to use NVE anyway, I tried it out and it was actually worse than the base game

**Reply by OXP3573** [reply]
> > That's obviously not the effort of a mod developer but Rockstar dev input
> 
> No, not really. It's just a pool crash and it isn't that deep

Your increased pool and PackFileLimit Adjuster overcome the loading into GTA Online but after 1-2 minutes of being in Freemode it consistently does the same thing again and closes the game without an error box. I've tried PoolManager.asi to identify what in gameconfig.xml could be changed to fix it, but its returning a different value each time and UnknownPool. 

I am extremely close to getting there, you might not think it is worth it but on my rig it looks pretty good. Functionality wise I can't see any conflict other than it exceeding the base limits set by Rockstar. The fact the crash happens consistently after the same time rather than at random means it is not a local problem with the memory, the fix would be harder if it was crashing at random intervals. Any suggestions?
